admin/class-metabox.php

```diff
@@ -648,6 +648,21 @@ function do_meta_box( $meta_field_def, $key = '' ) {
 			$placeholder = $meta_field_def['placeholder'];
 		}
 
+		// When choosen key is set add chosen class and determine which placeholder should shown.
+		if ( ! empty( $meta_field_def['chosen'] ) ) {
+			$class .= ' yoast-chosen';
```

Comment:
is there a reason there's a space before the class name within the string? Shouldn't this be `yoast-chosen` instead of ` yoast-chosen'?